src/main/java/org/jboss/msc/service/ServiceControllerImpl.java

```diff
@@ -486,14 +486,11 @@ void transition(final ArrayList<Runnable> tasks) {
                     }
                     break;
                 case LAZY: {
-                    if (state.getState() == State.UP && state != Substate.STOP_REQUESTED) {
```

Comment:
I think we don't want to change this; see https://issues.jboss.org/browse/MSC-169?focusedCommentId=13421096&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13421096